inc/template-functions.php

```diff
@@ -143,6 +143,24 @@ function hale_sidebar_location( $sidebar ) {
 
 function hale_get_branding_class() {
 
+	$css_file_path = "./wp/wp-includes/css/MjB-test.css";
```

Comment:
This is the location for my test CSS file - it seems to work.  This is the Hale repository writing into the JOTW repository.  Is this okay?

Reply:
As mentioned I would look at moving this to s3 bucket/uploads folder. Mainly also because that would also mean theres only one true css file for each site to use. If the css files lives in the theme or includes folder each container running on AWS will have its own css file. Which could cause issues with some not being updated.

Reply:
I would also suggest we build the css file to a temp css file then once file is written maybe rename the css file to the right name for the site...just means if the css build does have an error the users wont see a half styled site.